packages/get-packages/src/index.ts

```diff
@@ -1,3 +1,7 @@
+// This is a modified version of the package-getting in bolt
+// It supports yarn workspaces as well, and can fall back through
+// several options
```

Comment:
I feel that this has been refactored so much lately that this comment is no longer applicable
```suggestion
```